packages/app/src/cli/services/dev/tunnel-mode.ts

```diff
@@ -64,19 +63,6 @@ export async function getTunnelMode({
     requestedPort,
     actualPort,
     provideCertificate: async (appDirectory) => {
```

Comment:
We don't really need this to be part of the NoTunnel interface no? It's only used in one place and after validating that we are using localhost.

On line 346 of `services/dev.ts` you can just call `await generateCertificate` directly no?

Reply:
Heh, yep/. Updated 👍  Thanks Isaac